cli/lib/cli.js

```diff
@@ -173,7 +187,20 @@ module.exports = {
       .catch(util.logErrorExit1)
     })
 
-    logger.log()
+    program
+    .command('cache')
+    .usage('[command]')
+    .description('Manage the Cypress binary cache')
```

Comment:
Can you update 'Manage' to 'Manages' to keep in line with other commands having plural verb descriptors, ie: 'Opens ...',  'Installs ...',  'Verifies ...'?